src/pages/signin/ChangeExpensifyLoginLink.js

```diff
@@ -31,15 +32,17 @@ const defaultProps = {
 const ChangeExpensifyLoginLink = (props) => (
     <View style={[styles.changeExpensifyLoginLinkContainer, styles.mt3]}>
         {!_.isEmpty(props.credentials.login) && <Text style={styles.mr1}>{props.translate('loginForm.notYou', {user: props.formatPhoneNumber(props.credentials.login)})}</Text>}
-        <TouchableOpacity
+        <PressableWithFeedback
             style={[styles.link]}
             onPress={props.onPress}
+            accesibiltyRole="link"
```

Comment:
```suggestion
            accessibilityRole="link"
```

Reply:
nice catch!

Reply:
Woah, great catch 👀  committing this and resolving this convo